galaxy.yml

```diff
@@ -5,7 +5,7 @@ authors:
   - willthames (https://github.com/willthames)
   - Akasurde (https://github.com/akasurde)
 dependencies:
-  kubernetes.core: '>=5.0.0'
+  kubernetes.core: '<6.0.0'
```

Comment:
The collection has a dependency on `>=5.0.0`. If you add this new constraint, it should be something like `<6.0.0,>=5.0.0` or `>6.0.0`?

Reply:
Done!